README.md

```diff
@@ -40,19 +40,14 @@ PyKOALA has the following pre-requisites
 ```
 
 
-## Installation
+## Download
 
-To install, first clone the repository:
+To get PyKOALA, clone the repository, using git:
 
 ```
 git clone https://github.com/pykoala/koala.git
 ```
 
-Second part is installing it in your local machine with pip:
-
-```
-cd koala ; pip install . 
-```
```

Comment:
I guess including a brief statement on how to install the package (i.e. `cd /repository; python3 -m pip install .`) would help users with little experience on python.

Reply:
I agree

Reply:
Noted.

Reply:
OK

Reply:
Now that I think of it, perhaps we should include the activation of the virtual environment...

Reply:
Or, even better, the relevant section(s) in the documentation

Reply:
I'm not sure about overcrowding the README, especially if the venv information is already in the documentation

Reply:
I'll resolve for now, and modify README if we decided to include the venv later.